samples/nrf9160/nrf_cloud_coap_client/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Overlay to use nRF7002 EK on top of nrf9160 DK for Wi-Fi scanning
+
+# Actual configs for the Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+# Also see comments for CONFIG_HEAP_MEM_POOL_SIZE.
+CONFIG_NRF700X_SCAN_LIMIT=10
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_NATIVE=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+
+# Scan only using offload API
+CONFIG_WPA_SUPP=n
+CONFIG_MBEDTLS=n
+#CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=n
+#CONFIG_NET_UDP=n
+#CONFIG_NET_TCP=n
+CONFIG_NORDIC_SECURITY_BACKEND=n
+CONFIG_NET_IF_MAX_IPV4_COUNT=2
+CONFIG_NET_IF_MAX_IPV6_COUNT=2
+
+# Stack/heap tweaks needed to support Wi-Fi.
+# Wi-Fi shell sample has 25000 heap size. We need a bit more for location request purposes.
+# Heap allocation should be changed when CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT
+# and CONFIG_NRF700X_SCAN_LIMIT (which should be the same value) are changed.
+CONFIG_HEAP_MEM_POOL_SIZE=40000
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
This value is too low, it's set to 6144 in prj.conf.

Reply:
Good catch, thanks.